src/hiffy.rs

```diff
@@ -347,6 +368,20 @@ impl<'a> HiffyContext<'a> {
         Ok(())
     }
 
+    /// Blocking execution of a program, returning the results
+    pub fn execute_blocking(
```

Comment:
This looks really helpful to refactor it in this way -- in the spirit of less typing (and the really important stuff: fewer line breaks), what do you think about calling it `run` instead?

Reply:
Done!  How do you feel about renaming `execute` (which kicks off the program but doesn't block) to `start`, to avoid ambiguity with both `execute` and `run` functions?

Reply:
Yes, that sounds good!

Reply:
Done!